actors/evm/src/interpreter/precompiles/mod.rs

```diff
@@ -114,18 +114,18 @@ impl<RT: Runtime> Precompiles<RT> {
 #[derive(Debug)]
 pub enum PrecompileError {
     // EVM precompile errors
-    EcErr(CurveError),
+    EcErr,
     IncorrectInputSize,
     // FVM precompile errors
     InvalidInput,
     CallForbidden,
     TransferFailed,
-    VMError(ActorError),
+    VMError,
 }
 
 impl From<ActorError> for PrecompileError {
```

Comment:
I'd prefer to keep the error internals. I assume the complaint is that they're unused? In that case, we probably need to log them somewhere (will only affect actor debugging but it's still useful).